test/integration/targets/vmware_inventory/runme.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+[[ -n "$DEBUG" || -n "$ANSIBLE_DEBUG" ]] && set -x
+
+set -euo pipefail
+
+export ANSIBLE_CONFIG=ansible.cfg
+export vcenter_host="${vcenter_host:-0.0.0.0}"
+export vcenter_user="${vcenter_user:-user}"
+export vcenter_pass="${vcenter_pass:-pass}"
+VMWARE_CONFIG=test-config.vmware.yaml
```

Comment:
Adding `trap 'rm -f "${VMWARE_CONFIG}"' EXIT` should be all that is needed for automatic cleanup.

Reply:
Done.